### PR TITLE
chore: update issue template, move question to qa discussion

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <im@strrl.dev>

### What's changed and how it works?

Change the issue template, I think it could lead more clear to the usage and issue and discussion.

### Related changes

- [ ] This change also requires further updates to the [website](https://github.com/chaos-mesh/website) (e.g. docs)
- [ ] This change also requires further updates to the `UI interface`
- Need to **cheery-pick to release branches**
  - [ ] release-2.4
  - [ ] release-2.3
  - [ ] release-2.2

### Checklist

CHANGELOG

- [ ] I have updated the `CHANGELOG.md`
- [x] I have labeled this PR with "no-need-update-changelog"

Tests

- [ ] Unit test
- [ ] E2E test
- [x] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] **Breaking backward compatibility**

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```
[the remark this answers is not in the document]
Comment (STRRL) [reply .github/ISSUE_TEMPLATE/config.yaml:1]: +1

**Review by g1eny0ung — APPROVED** [review]
LGTM

**Comment by g1eny0ung** [comment]
@STRRL The license checker is failed. This file can be added to the ignore list.

**Comment by STRRL** [comment]
> @STRRL The license checker is failed. This file can be added to the ignore list.

Updated!

**Review by FingerLeader — APPROVED** [review]
LGTM

**Pull request merged**